pkg/ui/workspaces/db-console/src/views/databases/indexDetailsPage/redux.ts

```diff
@@ -63,7 +62,7 @@ export const mapStateToProps = createSelector(
     hasAdminRole,
     timeScale,
   ): IndexDetailsPageData => {
-    const stats = indexStats[generateTableID(database, table)];
+    const stats = indexStats[util.generateTableID(database, table)];
```

Comment:
Same with this mapStateToProps, but up to you!

Reply:
I'll address this in the connected table details PR